prow/plugins/invalidcommitmsg/invalidcommitmsg_test.go

```diff
@@ -165,15 +271,21 @@ Instructions for interacting with me using PR comments are available [here](http
 			}
 
 			comments := fc.IssueCommentsAdded
-			if len(comments) == 0 && tc.addedComment != "" {
-				t.Errorf("Expected comment with body %q to be added, but it was not", tc.addedComment)
+			if len(comments) != len(tc.addedComments) {
+				t.Errorf("Expected %v comments, but received %v", len(tc.addedComments), len(comments))
 				return
 			}
-			if len(comments) > 1 {
-				t.Errorf("did not expect more than one comment to be created")
-			}
-			if len(comments) != 0 && comments[0] != tc.addedComment {
-				t.Errorf("expected comment to be \n%q\n but it was \n%q\n", tc.addedComment, comments[0])
+
+			if len(comments) != 0 && !reflect.DeepEqual(comments, tc.addedComments) {
```

Comment:
This might be easier to express and debug by diffing it via `cmp`, i.E.
```
import github.com/google/go-cmp/cmp

if diff := cmp.Diff(comments, tc.addedComments); diff != "" {
 t.Errorf("Actual comments differ from expected comments: %s", diff)
}
```